src/user-research/getting-informed-consent-for-user-research.md

```diff
@@ -57,9 +57,9 @@ Whichever approach you take, make sure your consent materials and processes foll
 
 ### Creating a research information and consent sheet
 
-For each research activity you do, create a specific information and consent sheet that describes the research and gives the participant the information they need.
+For each research activity we do, cwe reate a specific information and consent sheet that describes the research and gives the participant the information they need.
```

Comment:
we instead of cwe - or is it github not updating things?

Reply:
A typo by me. I'll make sure to fix it.